exercises/security-rnd/software-engineer/README.md

```diff
@@ -20,20 +20,23 @@ You can obtain package data through the npm registry: `https://registry.npmjs.or
 
 #### Things to consider
 
-  1. Look at the inner "dependencies" object for analysis of first-order dependencies.
+  1. Look at the inner "dependencies" object for analysis of its dependencies and do so recursively.
   2. There are currently over 800K packages on npmjs.com, and the number is growing all the time.
   3. The packages update from time to time, just as their dependencies.
   4. What makes a good web service? API, architecture, data storage, low latency, scalability, monitoring, you name it :)
   5. Consider the quality and structure of your codebase; is it maintainable?
-  6. Consider production readiness (to some extent) and is it safe to deploy changes?
+  6. Consider production readiness (to some extent) 
```

Comment:
I'm not opposed to removing to `is it safe to deploy changes?`. But is there a strong reason for doing it?

Reply:
For me consider production readiness in the context of this home work is: make your application in a way that it can be packaged and delivered easily: docker, virtualenv, etc...

How would you evaluate that it is "safe to deploy changes" in the context of this home work?

Reply:
Yeah I think thats a fair point - I can see how it can be confusing. I wonder what the original reason this was included was. Maybe @tomersnyk knows?

Reply:
Happy to remove this part, but lets also remove the `to some extent` comment in that case:
```suggestion
  6. Consider production readiness
```